From 2643deb17cda7e0857fce1488f1bc7d5ca04cd12 Mon Sep 17 00:00:00 2001
From: Benjamin Berg <bberg@redhat.com>
Date: Tue, 25 Aug 2020 15:01:56 +0200
Subject: [PATCH 099/134] adaptive: Fix 64bit printing on non-64bit machines

(cherry picked from commit 0710804c2c2747839231142c6be95485ed1707b7)
Signed-off-by: Paolo Pisati <paolo.pisati@canonical.com>
(cherry picked from commit 506ab39fca8167f0a9604848eafc1cd593e1ac97)
Signed-off-by: Paolo Pisati <paolo.pisati@canonical.com>
---
 src/thd_engine_adaptive.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/thd_engine_adaptive.cpp b/src/thd_engine_adaptive.cpp
index acae7d4..8d33153 100644
--- a/src/thd_engine_adaptive.cpp
+++ b/src/thd_engine_adaptive.cpp
@@ -183,7 +183,7 @@ void cthd_engine_adaptive::dump_apat()
 	thd_log_info("..apat dump begin.. \n");
 	for (unsigned int i = 0; i < targets.size(); ++i) {
 		thd_log_info(
-				"target_id:%lu name:%s participant:%s domain:%d code:%s argument:%s\n",
+				"target_id:%" PRIu64 " name:%s participant:%s domain:%d code:%s argument:%s\n",
 				targets[i].target_id, targets[i].name.c_str(),
 				targets[i].participant.c_str(), (int)targets[i].domain,
 				targets[i].code.c_str(), targets[i].argument.c_str());
-- 
2.27.0

